extras/indexing/src/main/java/org/apache/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -127,6 +127,7 @@
     public static final String USE_PCJ_UPDATER_INDEX = "sc.use.updater";
 
     public static final String FLUO_APP_NAME = "rya.indexing.pcj.fluo.fluoAppName";
+    public static final String USE_PCJ_FLUO_UPDATER = "rya.indexing.pcj.updater.fluo";
     public static final String PCJ_STORAGE_TYPE = "rya.indexing.pcj.storageType";
```

Comment:
Why are you sneaking an extra Fluo config parameter in here?  Are you doing something with the Fluo App?